lib/logstash/environment.rb

```diff
@@ -114,6 +95,42 @@ def bundler_setup!(options = {})
       ::Bundler.setup
     end
 
+    def runtime_jars_root(dir_name, package)
+      ::File.join(dir_name, package, "runtime-jars")
+    end
+
+    def test_jars_root(dir_name, package)
+      ::File.join(dir_name, package, "test-jars")
+    end
+
+    def load_runtime_jars!(dir_name="vendor", package="jar-dependencies")
```

Comment:
the code of `load_runtime_jars!` and `load_test_jars!` is the same, it should be put in a common method, leaving them as 

```ruby
def load_runtime_jars!(dir_name="vendor", package="jar-dependencies")
  generic_load_jars!(runtime_jars_root(dir_name, package))
end

def load_test_jars!(dir_name="vendor", package="jar-dependencies")
  generic_load_jars!(test_jars_root(dir_name, package))
end
```